Makefile

```diff
@@ -18,7 +18,10 @@ lint:
 	golint -set_exit_status ./...
 
 build_proto: install
-	protoc --twirp_out=. --go_out=. --twirp_typescript_out=package_name=haberdasher:./example/ts_client ./example/service.proto
+	protoc --twirp_out=. --go_out=. --twirp_typescript_out=package_name=haberdasher:./example/ts_client ./example/*.proto
```

Comment:
So 👍 for automating more bits of this.  

Let's go ahead and do some further refactoring then:

1. Having build_proto just generate the go code
2. Add individual targets for building each client

I think this will make it clearer what each make target is doing.

Reply:
@larrymyers revised. I went ahead and cleaned up the Makefile a bit, hope that's OK! Any issues let me know.